daq/dhcp_monitor.py

```diff
@@ -0,0 +1,85 @@
+"""Encapsulate DHCP monitor/startup"""
+
+import logging
+import re
+import time
+
+from clib.tcpdump_helper import TcpdumpHelper
```

Comment:
Is the clib here the same as https://github.com/faucetsdn/faucet/tree/master/clib

Wondering how the user install the lib (directory lay out etc.). We should document that in one of the docs/*md file.

Also if TcpdumpHelper is a class you may want to import tcpdump_helper to meet the style guide.